lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -195,6 +194,9 @@ def main():
         module.params['folder'] = '/vm%(folder)s' % module.params
     module.params['folder'] = module.params['folder'].rstrip('/')
 
+    if module.params['name'] is None and module.params['uuid'] is None:
```

Comment:
Could you please use `required_one_of` just like [here](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/systemd.py#L313) ?